mail/protocols/imap/src/test/java/com/fsck/k9/mail/store/imap/ImapStoreTest.java

```diff
@@ -60,16 +60,6 @@ public void getFolder_shouldReturnImapFolderInstance() throws Exception {
         assertEquals(ImapFolder.class, result.getClass());
     }
 
-    @Test
-    public void getFolder_calledTwice_shouldReturnFirstInstance() throws Exception {
-        String folderName = "Trash";
-        ImapFolder imapFolder = imapStore.getFolder(folderName);
-
-        ImapFolder result = imapStore.getFolder(folderName);
-
-        assertEquals(imapFolder, result);
```

Comment:
Fun fact: This didn't work as intended because `assertEquals()` was used instead of `assertSame()`.